frankenphp.go

```diff
@@ -278,7 +279,7 @@ func calculateMaxThreads(opt *opt) error {
 // Init starts the PHP runtime and the configured workers.
 func Init(options ...Option) error {
 	if requestChan != nil {
-		return AlreadyStartedError
+		return AlreaydStartedError
```

Comment:
To revert

Reply:
Got up to date with main and this resolved itself

Reply:
Or wait, weird, how come the line is shown correctly on my local git, but out of sync here on github?

Reply:
Weird. Maybe should you squash all your commits and rebase against main?

Reply:
Rebasing resolved it 👍